src/python/pants/fs/fs_test.py

```diff
@@ -9,80 +9,64 @@
 from pants.engine.goal import Goal, GoalSubsystem
 from pants.engine.rules import Get, goal_rule, rule
 from pants.fs.fs import is_child_of
-from pants.testutil.test_base import TestBase
-
-
-class WorkspaceGoalSubsystem(GoalSubsystem):
```

Comment:
This class, and all the below rules / dataclasses, get inlined into the single unit test that use it. I figured this results in better namespacing.